enterprise/backend/src/metabase_enterprise/data_editing/describe.clj

```diff
@@ -31,10 +31,10 @@
    [:parameters [:sequential ::describe-param]]])
 
 (defn- param-value
-  [param-mapping row-delay]
-  (case (:sourceType param-mapping)
-    "constant" (:value param-mapping)
-    "row-data" (when row-delay (get @row-delay (keyword (:sourceValueTarget param-mapping))))
+  [param-map row-delay]
```

Comment:
Param-map elsewhere refers to the map of all parameters, where this is a specific one. Maybe param-setting is a better name, to differentiate.